### PR TITLE
mysql support link

### PR DESCRIPTION
**Pull request by mabdullahadeel**
## Description

Added MySQL support to `cookiecutter-django`.

Checklist:

- [x] I've made sure that tests are updated accordingly (especially if adding or updating a template option)
- [x] I've updated the documentation or confirm that my change doesn't require any updates

## Rationale
Since these changes can't be merged to main `cookiecutter:master` due to various reasons, mentioned in #3346 and #3539 , I have added the link to the fork in `README.md`.

Closes #3539

### CODE REVIEW

**Comment by Afrowave** [comment]
Well done @mabdullahadeel.  
  
Thank you.

File: README.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ production-ready Django projects quickly.
 
 -   Only maintained 3rd party libraries are used.
 -   Uses PostgreSQL everywhere (10.19 - 14.1)
+    -  Now, MySQL is also supported. Please visit [this fork](https://github.com/mabdullahadeel/cookiecutter-django-mysql) to get full cookiecutter support for MySQL.
```
Comment (browniebroke) [inline README.md:53]:
```suggestion
-   Uses PostgreSQL everywhere: 10.19 - 14.1 ([MySQL fork](https://github.com/mabdullahadeel/cookiecutter-django-mysql) also available).
```

File: README.md [hunk 2]
```diff
@@ -203,6 +204,10 @@ If you are using [PyUp](https://pyup.io) to keep your dependencies updated and s
 
 Scattered throughout the Python and HTML of this project are places marked with "your stuff". This is where third-party libraries are to be integrated with your project.
 
+## For MySQL users
+To get full mysql support in addition to the default Postgresql, you can use this fork of the cookiecutter-django:
```
Comment (browniebroke) [inline README.md:208]:
```suggestion
To get full MySQL support in addition to the default Postgresql, you can use this fork of the cookiecutter-django:
```

**Comment by browniebroke** [comment]
Thanks a lot for this!

**Pull request merged**